hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/ECKeyOutputStream.java

```diff
@@ -51,6 +51,7 @@
  * block output streams chunk by chunk.
  */
 public class ECKeyOutputStream extends KeyOutputStream {
+  private final int maxRetriesOnStripeWriteFailures;
```

Comment:
I think this instance variable is not necessary, we use it just in one place, where we can also call config.getMaxECStripeWriteRetries() as well.

Reply:
Make sense. I have removed that var.